internal/pkg/cli/validate.go

```diff
@@ -178,6 +205,37 @@ func basicNameValidation(val interface{}) error {
 	return nil
 }
 
+func basicCronValidation(sched string) error {
+	every := "@every "
+	if strings.HasPrefix(sched, every) {
+		if err := basicDurationValidation(sched[len(every):], 60); err != nil {
+			if err == errDurationInvalid {
+				return fmt.Errorf("interval %s must include a valid Go duration string (example: @every 1h30m)", sched)
+			}
+			return fmt.Errorf("interval %s is invalid: %s", sched, err)
+		}
+	}
+	_, err := cron.ParseStandard(sched)
+	if err != nil {
+		return fmt.Errorf("schedule %s is invalid: %s", sched, errScheduleInvalid)
+	}
+	return nil
+}
+
+func basicDurationValidation(duration string, minDurationSecs int) error {
+	parsedDuration, err := time.ParseDuration(duration)
+	if err != nil {
+		return errDurationInvalid
+	}
+	if parsedDuration.Seconds() != float64(int64(parsedDuration.Seconds())) {
+		return errDurationBadUnits
+	}
+	if parsedDuration.Seconds() < float64(minDurationSecs) {
+		return fmt.Errorf("duration must be greater than %ds", minDurationSecs)
+	}
+	return nil
+}
```

Comment:
What do you think of this? https://play.golang.org/p/soKCp4v29S5

The `time.Duration` type is comparable I think this makes it slightly more readable as:
1.  The type of `min` becomes `time.Duration` 
2. The second if check is just `parsedDuration < 1 * time.Second`

Reply:
Hmm. But this doesn't error out for fractions of seconds greater than 1s, "1h55.5s" or "1h345ms", for instance. And "0m" gives the BadUnits/subsecond error rather than the too small error. 

What if we leave the float64/int64 conversion stuff (I can add a comment explaining what it's doing) but make the min-as-time.Duration change (which is great!)?